src/systems/input.rs

```diff
@@ -37,3 +76,8 @@ pub fn keyboard_input_system(
             .min((window.height / 2.0) - PADDLE_HEIGHT / 2.0);
     }
 }
+
+fn get_translation(time: f32) -> f32 {
```

Comment:
I had to graph this one to figure it out

Maybe just make a quick note that the real roots are 0 and 2, and that higher order even quadratics kinda look flatter and flatter

Reply:
Will do, I will probably change this back to `powi(5)` since that graph is a quintic with the shape we want, root at `0`. Was just messing about with these.